tests/state_test.go

```diff
@@ -727,8 +741,8 @@ func TestAllETH(t *testing.T) {
 
 func runETHTests(t *testing.T, fileNames []string, skipTests map[string]string) {
 	unsupportedForkConfigs := map[string]bool{
-		"Constantinople":               true,
-		"ConstantinopleFix":            true,
+		"Constantinople": true,
+		// "ConstantinopleFix":            true,
```

Comment:
Oh and if possible remove this just so there is no conflicts later on, this should never be added back in